docs/proposals/pod-pid-namespace.md

```diff
@@ -0,0 +1,58 @@
+# Shared PID Namespace for the Docker Runtime
+
+Pods share many namespaces, but the ability to share a PID namespace was not
+supported by Docker until version 1.12. This document proposes how to roll out
+support for sharing the PID namespace in the docker runtime.
+
+## Motivation
+
+Sharing a common PID namespace between containers is desirable for several
+reasons:
+
+  1. It enables signaling between containers, which is useful for side cars
```

Comment:
Linking https://github.com/kubernetes/kubernetes/issues/1615 could help since it has further discussion of why this makes sense for pods and of some of the history.